pkg/minikube/driver/driver.go

```diff
@@ -28,6 +28,8 @@ import (
 const (
 	// Mock driver
 	Mock = "mock"
+	// Kic driver
+	Kic = "kic"
```

Comment:
Go variables which reference initialisms should be consistently uppercase or lowercase, so `KIC` instead of `Kic`.

https://github.com/golang/go/wiki/CodeReviewComments#initialisms